frontend/src/components/Menu.vue

```diff
@@ -0,0 +1,27 @@
+<!-- TODO: Extract all unnecessary logic from App.vue to a separate view -->
```

Comment:
same here, we leave todo for smth that we need to refactor or finish (etc) in the future.

I see that you wanted to leave a comment so the changes are related to the task that you were working on.

But usually, we process every task in a **separate** pull request